cli/cfncluster/cfnconfig.py

```diff
@@ -268,47 +270,133 @@ def __init__(self, args):
             pass
 
         # Determine if EBS settings are defined and set section
+
+        # #self.parameters.append(("EBSSubstackURL",
+        #                         "https://s3.us-east-2.amazonaws.com/shuningc-cfncluster/extra_template/tropo_vol_substack.cfn.json"))
+
+        # CHANGED: adapt to list of ebs_settings
```

Comment:
This __init__ function is growing more and more and becoming very hard to read and impossible to test. It would be nice to move at the least the new code portions into separate functions which might be easily unit tested.
As a very general guideline: almost every time a comment is necessary to clarify the intent of the code maybe a function with a self-explanatory name can be created